src/main/java/com/redhat/cloud/policies/app/auth/RbacFilter.java

```diff
@@ -76,6 +78,9 @@ public void filter(ContainerRequestContext requestContext) throws IOException {
       if (warnSlowRbac.get() && (t2 - t1) > 500) {
         log.warning("Call to RBAC took " + (t2 - t1) + "ms");
       }
+      if (span != null) {
```

Comment:
`span` can be null?

Reply:
Probably not but there are null check on spans in the existing code so I chose the safest path 😄

Reply:
https://github.com/RedHatInsights/policies-ui-backend/blob/133e13a95e70a9f6ce403b7f52ef7382659c4028/src/main/java/com/redhat/cloud/policies/app/rest/PolicyCrudService.java#L1054-L1056

The Javadoc isn't really clear about whether or not `span` can be `null` when `tracer.buildSpan("getRBac").start()` is called.

Reply:
I removed the null-check.